web/src/api/ipfix.ts

```diff
@@ -118,7 +118,7 @@ export interface Fields {
   /** Network Events */
   NetworkEvents?: string[];
   /** Logical OR combination of unique TCP flags comprised in the flow, as per RFC-9293, with additional custom flags to represent the following per-packet combinations: SYN+ACK (0x100), FIN+ACK (0x200) and RST+ACK (0x400). */
-  Flags?: number;
+  Flags?: string;
```

Comment:
A string array will be more consistent with other implementations (network events, interfaces directions etc)

See https://github.com/netobserv/flowlogs-pipeline/pull/747#discussion_r1839867274

Reply:
done https://github.com/netobserv/network-observability-console-plugin/pull/632/commits/9407e7d543fca78b3d2b48573fe8cc26ac67cd28